tests/Unit/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/Test_KerrSchild.cpp

```diff
@@ -0,0 +1,299 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Index.hpp"
+#include "DataStructures/Tensor/EagerMath/Magnitude.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/GrTags.hpp"
+#include "Options/Options.hpp"
+#include "Options/ParseOptions.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/KerrSchild.hpp"
+#include "tests/Unit/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/VerifyEinsteinSolution.hpp"
+#include "tests/Unit/PointwiseFunctions/GeneralRelativity/GrTestHelpers.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+
+template <typename DataType>
+tnsr::I<DataType, 3, Frame::Inertial> spatial_coords(
+    const DataType& used_for_size) noexcept {
+  auto x = make_with_value<tnsr::I<DataType, 3, Frame::Inertial>>(used_for_size,
+                                                                  0.0);
+  get<0>(x) = 1.99;
+  get<1>(x) = -0.01;
+  get<2>(x) = 0.01;
+  return x;
+}
+
+template <typename DataType>
+void test_schwarzschild(const DataType& used_for_size) noexcept {
+  // Schwarzschild solution is:
+  // H        = M/r
+  // l_mu     = (1,x/r,y/r,z/r)
+  // lapse    = (1+2M/r){-1/2}
```

Comment:
there is something missing between the `)` and `{`, maybe a `^` or a `/`? and next line too